fsharp-backend/src/BackendOnlyStdLib/HttpClient.fs

```diff
@@ -93,6 +93,8 @@ let convertHeaders (headers : AspHeaders) : HttpHeaders.T =
 exception InvalidEncodingException of int
 
 // CLEANUP add dark-specific user-agent
+/// Uses an internal .NET HttpClient to make a requst
+/// and process response into a Dark HttpResult response
```

Comment:
```suggestion
/// and process response into a HttpResult response
```

I think it isn't helpful to add "Dark" in comments, as everything is Dark. In this case, HttpResult is obviously a type.